mParticle-Apple-SDK/MPBackendController.m

```diff
@@ -379,7 +379,7 @@ - (void)processDidFinishLaunching:(NSNotification *)notification {
     if ([[UIDevice currentDevice].systemVersion floatValue] >= 8.0) {
         NSUserActivity *userActivity = userInfo[UIApplicationLaunchOptionsUserActivityDictionaryKey][@"UIApplicationLaunchOptionsUserActivityKey"];
         
-        if (userActivity) {
+        if (userActivity.webpageURL) {
```

Comment:
If `stateMachine.launchInfo` has more information in it than just information based on a `webpageUrl`, I think we should create a defaultUrl of empty string so that there is still some valuable information in `launchInfo`.  

I saw in other parts of the code something like this:

https://github.com/mParticle/mparticle-apple-sdk/blob/590fc9a21ef26af8364a0489b6ec3d6c627683d9/mParticle-Apple-SDK/AppNotifications/MPAppNotificationHandler.m#L284-L289

Reply:
But if we pass in an empty string url and a nil options there isn't any valuable info in launchOptions. Its just initialized with all nil values and doesn't get any other data on its own.